issue_release_status.go

```diff
@@ -31,12 +56,95 @@ func (handler *PRCreateHandler) Handle(ctx context.Context, eventType, deliveryI
 	installationID := githubapp.GetInstallationIDFromEvent(&event)
 
 	ctx, logger := githubapp.PreparePRContext(ctx, installationID, repository, prNum)
-
-	logger.Debug().Msgf("Event action is %s", event.GetAction())
 	if event.GetAction() != "created" {
 		return nil
 	}
 
+	prBase := event.GetPullRequest().GetBase().GetRef() // The branch which the pull request is ending.
+
+	// Retrieve auto-release-policy
+
+	serviceName := event.GetRepo().GetName()
+	serviceName = strings.TrimSuffix(serviceName, "-service")
```

Comment:
We could down the road consider making these configurable as well, as this is created in the open.